bdk_core/src/sparse_chain.rs

```diff
@@ -547,7 +547,7 @@ impl SparseChain {
     }
 
     // Invalidate all checkpoints after the given height
-    fn invalidate_checkpoint(&mut self, height: u32) {
+    pub fn invalidate_checkpoint(&mut self, height: u32) {
```

Comment:
This method is intentionally private. You invalidate things exclusively with a checkpoint.